docroot/themes/humsci/su_humsci_admin/js/node-protection.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file
+ * Stops page from changing when user is posting.
+ */
+
+(function ($, Drupal) {
+  'use strict';
+  // Allow Save Submit button.
+  var save = false;
+  // Flag the form has been edited.
+  var edit = false;
+
+  Drupal.behaviors.nodeEditProtection = {
```

Comment:
Are behaviors supposed to be namespaced with the module name? I can't find anything in the docs specifying a convention on this.

Reply:
Yes, they are. i went through the API today